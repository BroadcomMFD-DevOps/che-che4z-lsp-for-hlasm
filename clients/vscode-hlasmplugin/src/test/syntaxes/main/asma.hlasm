LABEL    INSTR OP THIS LINE SHOULD NOT BE AFFECTED
** ASMA123I - This is just a remark in general hlasm grammar
** ASMA123E - This is just a remark in general hlasm grammar
LABEL    INSTR OP THIS LINE SHOULD NOT BE AFFECTED
