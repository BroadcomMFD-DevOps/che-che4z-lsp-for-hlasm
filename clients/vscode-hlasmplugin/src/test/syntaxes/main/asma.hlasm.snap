>LABEL    INSTR OP THIS LINE SHOULD NOT BE AFFECTED
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^^ source.hlasm punctuation.definition.tag.hlasm
#                 ^ source.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
>** ASMA123I - This is just a remark in general hlasm grammar
#^ source.hlasm comment.hlasm
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
>** ASMA123E - This is just a remark in general hlasm grammar
#^ source.hlasm comment.hlasm
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
>LABEL    INSTR OP THIS LINE SHOULD NOT BE AFFECTED
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^^ source.hlasm punctuation.definition.tag.hlasm
#                 ^ source.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
>